### ISSUE TITLE
Raspberry Pi + Ubuntu Server instalation enviroment error

### ISSUE THREAD

**Issue by okruhliak** [simplex-chat/simplexmq#1397]
Environment: Raspberry Pi 5/4GB, Ubuntu Server 24.10

Installation according to "git: Using your distribution" ended with an error:
E: Unable to locate package libffi7
E: Unable to locate package libncurses5
E: Unable to locate package libtinfo5

Ubuntu Server 24.10 is available
Package: libffi-dev Version: 3.4.6-1build1
Package: libncurses-dev Version: 6.4 20240113-1ubuntu2
Package: libtinfo6 Version: 6.4 20240113-1ubuntu2

Can I install the current versions instead of the recommended ones?

**Comment by DarkmatterUAE** [comment]
Yes, you can install libffi8 libncurses6 and libtinfo6, that would work, too. Have tested on Debian 12 and it worked.
